source/PlayerInfoPanel.cpp

```diff
@@ -809,6 +845,23 @@ void PlayerInfoPanel::DrawFleet(const Rectangle &bounds)
 			pos.Y() += 20.;
 		}
 	}
+	// Show tooltip that tells what is wrong with the ship under the mouse.
+	else if(error || warning)
+	{
+		constexpr int WIDTH = 250;
+		constexpr int PAD = 10;
```

Comment:
What about defining these two in interface.txt? Could be for another PR.

Reply:
Per the PR description, I think I might actually just pull the changes to the player info panel out. This is really just a new UI feature, not a bug fix.